src/dropdown/dropdown.ts

```diff
@@ -83,16 +86,26 @@ export class NgbDropdown {
   @Input('open') _open = false;
 
   /**
+   * Placement of a dropdown. Use "top-right" for dropups.
```

Comment:
Defintivelly. We are going to add appropriate type in a PR that introduces more positioning options (including `auto`).